### DISCUSSION TITLE
Unable to boot AMD clients

### DISCUSSION THREAD

**Discussion by JB-Prof** [ltsp/ltsp#473] [Q&A]
Hello,

We have some AMD computers we are trying to deploy on LTSP but they are hanging on the iPXE boot it seems.  I have attached a couple of photos.

The server is running Ubuntu Mate 18.04.5 LTS - we run many Intel machines off the same server for several years with no issues but not having success with the AMD ones.

Any recommendations?

Thank you so much,
JB

![Screenshot_1](https://user-images.githubusercontent.com/59401476/120510131-200af880-c397-11eb-9108-fd59891abae4.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/59401476/120510150-24371600-c397-11eb-91ac-56b4048ea886.jpg)
![Screenshot_3](https://user-images.githubusercontent.com/59401476/120510157-2600d980-c397-11eb-85b3-e90d8d5c1718.jpg)

**Comment by alkisg** [comment]
Hi, have a look at https://github.com/ltsp/ltsp/discussions/276#discussioncomment-102218.

**Reply by JB-Prof** [reply]
Thank you I will try this out and report back here.
